pkg/cloud/ovirt/clients/machineservice.go

```diff
@@ -120,6 +120,30 @@ func (is *InstanceService) InstanceCreate(
 		}
 	}
 
+	isAutoPinning := false
+	if providerSpec.AutoPinningPolicy != "" {
```

Comment:
why not:
providerSpec.AutoPinningPolicy != "" && autoPinningPolicy != ovirtsdk.AUTOPINNINGPOLICY_DISABLED

Reply:
as we talked offline, better to leave it as is and change the `isAutoPinning = true` placement.